<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QWidget" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="text">
      <string>Project - K</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold; font-size: 14pt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputLogin">
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputSenha">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Senha</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoEntrar">
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="botaoCadastrar">
     <property name="text">
      <string>Cadastrar-se</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelMensagem">
     <property name="text">
      <string/>
     </property>
     <property name="styleSheet">
      <string>color: red;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
